org.flathub.p3x-onenote.yml

```diff
@@ -0,0 +1,77 @@
+app-id: org.flathub.p3x-onenote
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - "--socket=wayland"
```

Comment:
This won't work on x11 session then, you need fallback-x11 too. If the app doesn't support native wayland, just use `--socket=x11`, then it will work on both x11 and wayland using xwayland.